03_launch_mgmt_cluster.sh

```diff
@@ -219,6 +224,37 @@ EOF
   popd
 }
 
+#
+# Launch and configure fakeIPA
+#
+function launch_fakeIPA() {
+  # Create a folder to host fakeIPA config and certs
+  mkdir -p /opt/metal3-dev-env/fake-ipa
+  if [[ "${EPHEMERAL_CLUSTER}" == "kind" && "${IRONIC_TLS_SETUP}" == "true" ]]; then
+    cp "${IRONIC_CACERT_FILE}" "/opt/metal3-dev-env/fake-ipa/ironic-ca.crt"
+  elif [[ "${IRONIC_TLS_SETUP}" == "true" ]]; then
+    # wait for ironic to be running to ensure ironic-cert is created 
+    kubectl -n baremetal-operator-system wait --for=condition=available deployment/baremetal-operator-ironic --timeout=900s
+    # Extract ironic-cert to be used inside fakeIPA for TLS 
+    kubectl -n legacy get secret -n baremetal-operator-system ironic-cert -o json -o=jsonpath="{.data.ca\.crt}" | base64 -d > /opt/metal3-dev-env/fake-ipa/ironic-ca.crt
+  fi
+  # Create fake IPA custom config
+  cat << EOF > "${WORKING_DIR}/fake-ipa/config.py"
+FAKE_IPA_API_URL = "https://${CLUSTER_BARE_METAL_PROVISIONER_IP}:${IRONIC_API_PORT}"
+FAKE_IPA_INSPECTION_CALLBACK_URL = "${IRONIC_URL}/continue_inspection"
+FAKE_IPA_ADVERTISE_ADDRESS_IP = "${EXTERNAL_SUBNET_V4_HOST}"
+FAKE_IPA_INSECURE = ${FAKE_IPA_INSECURE:-False}
+FAKE_IPA_CAFILE = "${FAKE_IPA_CAFILE:-/root/cert/ironic-ca.crt}"
```

Comment:
Why under `/root` ?

Reply:
Just following the current structure of vbmc and sushytools

Reply:
To add to that, I think the current structure is because these are used inside containers and those containers must likely use the root user.